src/raw.ts

```diff
@@ -840,6 +840,12 @@ export interface RawRemarkableApi {
     metadata: Metadata,
   ): Promise<[RawFileEntry, Promise<void>]>;
 
+  /**
+   * create a list hash from a set of entries
+   * @param entries - the entries to hash
+   */
+  makeListHash(entries: RawEntry[]): string;
```

Comment:
I don't think there's much utility in exposing this, and it doesn't rely on the state of raw, so I'd move to a utility function. It could still be placed within this file, I just wouldn't put it in the RawApi.

That said, I wouldn't make any changes until there's agreement on the rootHash, as this would be moot.